charts/dsp-argocd/values.yaml

```diff
@@ -0,0 +1,90 @@
+argo-cd:
+  global:
+    image:
+      tag: v1.7.7 # TODO this must be removed the next time we update ArgoCD
+  server:
+    extraArgs:
+    - --insecure # Required to enable TLS termination at Ingress/LB
+    config: # SSO
+      url: https://ap-argocd.dsp-devops.broadinstitute.org
+      dex.config: |
+        connectors:
+          - type: github
+            id: github
+            name: GitHub
+            config:
+              clientID: 44eb2e55b99be9fef021
+              clientSecret: $dex.github.clientSecret
+              orgs:
+              - name: DataBiosphere
+              - name: broadinstitute
+      accounts.terra-ci: apiKey
+      accounts.fc-jenkins: apiKey
+      accounts.fcprod-jenkins: apiKey
+      accounts.local-notsuitable: login
+      accounts.local-suitable: login
+      configManagementPlugins: |
+        - name: legacy-configs
+          generate:
+            command: ["legacy-configs.sh"]
+        - name: helmfile
+          init:
+            command: ["helmfile.sh", "init"]
+          generate:
+            command: ["helmfile.sh", "generate"]
+        - name: helm-values
+          init:
+            command: ["helm-values.sh", "init"]
+          generate:
+            command: ["helm-values.sh", "generate"]
+        - name: terra-helmfile-argocd
+          generate:
+            command: ["terra-helmfile-argocd.sh", "generate"]
+        - name: terra-helmfile-app
+          generate:
+            command: ["terra-helmfile-app.sh", "generate"]
+    service:
+      type: NodePort # Required to enable forwarding from Ingress/LB to service
```

Comment:
Not trying to reinvent the wheel here, but should we switch to cloud native loadbalancing for this too?

Reply:
Hmm... Created a ticket: https://broadworkbench.atlassian.net/browse/DDO-940

Reply:
I don't consider it high priority, because ArgoCD isn't prod in the sense that it's serving requests from Terra users